Update Android stores (#425)
sailfishos/docs.sailfishos.org: Support/Help_Articles/Android_App_Support/README.md

If you would like to remove Android apps (and support) from your Sailfish device, read [**this help article**](/Support/Help_Articles/Android_App_Support/Removing_Android_App_Support/).

# Installing Android applications on your Sailfish device
Installing Android applications is done by downloading **apk** files (an Android package file used to distribute applications) from Android application stores, followed by the installation in the app store.

Before installing any Android apps, **Android AppSupport** (simulation of the Android system on Sailfish OS) must be installed. It is available in the Jolla Store. This requires the Sailfish OS licence.

Android apps are installed in the internal storage of a phone. Installing apps on SD cards is not supported.

> After installing Android apps, the folder ```/home/.appsupport/instance/defaultuser/data/data``` has all of your installed Android apps. The folder ```/home/defaultuser/android_storage/Android/data/``` contains the data meaningful to you in various Android apps.

Reviewer: ```suggestion
After installing Android apps, the folder ```/home/.appsupport/instance/defaultuser/data/data``` has all of your installed Android apps. The folder ```/home/defaultuser/android_storage/Android/data/``` contains the data meaningful to you in various Android apps. Files saved using Android apps, for example downloaded files in a web browser app or photos taken in a camera app, are stored in their respective folders in ```/home/defaultuser/android_storage```.
```

Author: Done.

## Installing applications from application stores

> The SailfishOS operating system of Jolla supports Android apps that are featured in the Jolla Store and Jolla-certified app partner stores.

Reviewer: ```suggestion
Sailfish OS supports Android apps featured in Jolla Store and the Jolla-certified app partner stores.
```

Slightly off-topic, but `SailfishOS` is used here and on line 31, whereas `Sailfish OS` is used elsewhere.

Author: Done.
